### ISSUE TITLE
Bad pronunciation in Basque

### ISSUE THREAD

**Issue by Giovani93** [espeak-ng/espeak-ng#1188]
Hello.
Please, if It is false, just edit me.
I have noticed a bad pronounciation of letter 's' in Basque. This letter is pronounced as palatalised s, not alveolar.
For example, like 'ship' in English.
Please, could You edit It?
Thank You.
Marco

**Comment by sukiletxe** [comment]
This is not correct. S and z are both alveolar, and x is palatal (like English ship as you said).

**Comment by hozosch** [comment]
I guess you mean that retracted s, which is difficult to do in ESpeak I guess. This is also the issue Greek and Spanish are facing.

**Comment by sukiletxe** [comment]
I think s, z and x are pronounced correctly.
